app/src/main/res/layout/activity_zim_host.xml

```diff
@@ -5,9 +5,18 @@
   android:layout_width="match_parent"
   android:layout_height="match_parent"
   android:fitsSystemWindows="true"
-  tools:context=".webserver.ZimHostActivity">
+  tools:context=".webserver.ZimHostFragment">
 
-  <include layout="@layout/layout_standard_app_bar" />
+  <androidx.appcompat.widget.Toolbar
```

Comment:
Are you not using the include of the toolbar to avoid using `findViewById` in kotlin?

Reply:
getting unresolved reference if I use and import `kotlinx.android.synthetic.main.layout_toolbar.toolbar` instead

Reply:
yes, that is the multi module problem, the thing is using `findViewById` or doing the workaround for exporting the synthetic property is waaaaay less hassle then creating X*NumberOfScreens toolbars and managing these specific stupid attributes they have. Maybe a style would work also? I remember pulling my hair out getting these things working in Dark mode in a repeatable way

Reply:
hmm, a style is a good idea. Tried using findViewById but the bar is empty.

Reply:
when in a fragment you need to call findViewById on the view, not the activity incase that was the error

Reply:
that was the error! I think

Reply:
yes!